forge-gui/res/cardsfolder/upcoming/witch_king_sky_scourge.txt

```diff
@@ -0,0 +1,15 @@
+Name:Witch-king, Sky Scourge
+ManaCost:5 B R
+Types:Legendary Creature Wraith Noble
+PT:5/5
+K:Flying
+K:Undying
+T:Mode$ AttackersDeclared | ValidAttackers$ Wraith | AttackingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever you attack with one or more Wraiths, exile the top X cards of your library, where X is their total power. You may play those cards this turn.
+SVar:TrigExile:DB$ Dig | Defined$ You | DigNum$ X | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STPlay | ForgetOnMoved$ Exile | RememberObjects$ RememberedCard | SubAbility$ DBCleanup
+SVar:STPlay:Mode$ Continuous | EffectZone$ Command | Affected$ Card.IsRemembered | MayPlay$ True | AffectedZone$ Exile | Description$ you may play those cards this turn
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Count$SumPower_Wraith.attacking+YouCtrl
```

Comment:
which Wraiths are checked is determined when it triggers (in case some get removed in response)
so you need to use TriggeringObject

Reply:
Any cards I should look at ?

Reply:
Try `TriggeredAttackers$CardPower`

Reply:
Hmm I think this makes it crash 

![image](https://github.com/Card-Forge/forge/assets/67333662/9587aa48-c9c4-42ce-b2cd-53df4a082d68)

Reply:
working now, thanks